config/initializers/new_framework_defaults_6_1.rb

```diff
@@ -0,0 +1,67 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Do we need this file, or can it be deleted?

Reply:
I missed this one, have deleted now